snappy_wrappers/wrappers/pvactools/pvacseq/environment.yaml

```diff
@@ -0,0 +1,15 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+  - default
+dependencies:
+  - python==3.6.15
```

Comment:
Now that `pvactools` is included as a docker container, can we move python to 3.12?